xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/macros.vm

```diff
@@ -306,6 +306,13 @@
                 </td>
               #elseif(("$!colprop.type" == 'text' || "$!colprop.type" == 'number') && $isFilterable)
                 <td class="xwiki-livetable-display-header-filter"><input id="xwiki-livetable-${htmlLiveTableId}-filter-${velocityCount}" name="${colname}" type="text" #if("$!colprop.size" != '') size="${colprop.size}" title="$escapetool.xml($services.localization.render('platform.livetable.filtersTitle', [$services.localization.render("${transprefix}${colname}")]))"#end /></td>
+              #elseif("$!colprop.type" == 'date')
+                #set($ok = $xwiki.jsfx.use('js/xwiki/table/livetabledate.js', true))
+                <td class="xwiki-livetable-display-header-filter">
+                  ## Hidden input for date fields allows us to send timestamps to the server while displaying pretty dates
+                  <input id="xwiki-livetable-${htmlLiveTableId}-filter-${velocityCount}" name="${colname}" type="hidden" aria-hidden="true" />
```

Comment:
Yup, I agree. Just checked the specification, type="hidden" hides it from the accessibility tree in all browsers so it won't be shown no matter what.